v3/zoned_client.go

```diff
@@ -0,0 +1,135 @@
+package v3
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/exoscale/egoscale/v3/oapi"
+)
+
+const (
+	DefaultHostPattern = "https://api-%s.exoscale.com/v2"
+
+	EnvKeyAPIEndpointPattern = "EXOSCALE_API_ENDPOINT_PATTERN"
+	EnvKeyAPIEndpointZones   = "EXOSCALE_API_ENDPOINT_ZONES"
+)
+
+var (
+	// DefaultZones list (available in oapi code).
+	// When new zone is added or existing removed this slice needs to be updated.
```

Comment:
I wish we could generate this somehow. Is there really no way?

Reply:
We can generate it, it should be fairly straightforward. I'll create a separate story for that.